paper/paper.Rmd

```diff
@@ -1,35 +1,42 @@
 ---
-title: "datawizard: An R Package for Easy Data Wrangling"
+title: "datawizard: An R Package for Easy Data Wrangling and Transformations"
```

Comment:
I am not fully sold on the terms I am using here: Data Wrangling and Data Transformations.

Basically, we should distinguish between two key categories in which _datawizard_ functions can be broken down:
one that deal with wrangling and the other that deal with statistical transformations. 

But I am not sure what other term to use here, or should we just give up on trying to force the section to have "data" prefix?

In that case, this would be: 
> datawizard: An R Package for Easy Data Wrangling and Statistical Transformations